### PR TITLE
CSP-865 Created area of interest edit page

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

File: src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    public async Task<EditAreaOfInterestViewModel> GetAreaOfInterests(string employerAccountId, CancellationToken cancellationToken)
+    {
+        EditAreaOfInterestViewModel editAreaOfInterestViewModel = new EditAreaOfInterestViewModel();
+        var memberId = Guid.Parse(_sessionService.Get(Constants.SessionKeys.MemberId)!);
+        var memberProfiles = await _outerApiClient.GetMemberProfile(memberId, memberId, false, cancellationToken);
+        var profilesResult = await _outerApiClient.GetProfilesByUserType(MemberUserType.Employer.ToString(), cancellationToken);
+
+        editAreaOfInterestViewModel.FirstSectionInterests = SelectProfileViewModelMapping(profilesResult.Profiles.Where(x => x.Category == "ReasonToJoin").ToList(), memberProfiles.Profiles);
```
Comment (MarkFCain) [inline src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs:72]: "ReasonToJoin" should be a shared constant somewhere, and all references to it should use that constant

File: src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        editAreaOfInterestViewModel.SecondSectionInterests = SelectProfileViewModelMapping(profilesResult.Profiles.Where(x => x.Category == "Support").ToList(), memberProfiles.Profiles);
```
Comment (MarkFCain) [inline src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs:74]: "Support" should be a shared constant somewhere, and all references to it should use that constant (not the string "Support")

Check the views use this constant if they are using "Support" inline or for logical processing

**Review by MarkFCain — DISMISSED** [review]

File: src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs (hunk 1 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs]: Done

File: src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs (hunk 2 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Employer.Aan.Web/Controllers/EditAreaOfInterestController.cs]: Done

**Pull request closed without merging**